frontend/src/components/Nav/Page/RenderComponentScroll.tsx

```diff
@@ -25,6 +25,7 @@ const componentStyle = kialiStyle({
 
 interface Props {
   className?: any;
+  noScroll?: boolean;
```

Comment:
This noScroll should have the same behavior as globalScrollbar? I think it could have then the same or similar prop name, and then, see https://github.com/kiali/kiali/pull/8316/files#diff-0ccf5e7a42d9b1b205ba5ba64983ee37daf2dd084d0545b45c64f9a966ed0ec4R70

Reply:
Yes totally....My bad I did so changes and the final result is like you said... not neccesary